www/layouts/course_collections/single.html

```diff
@@ -0,0 +1,27 @@
+{{ define "main" }}
+{{- $collection := . -}}
+<div>
+  {{ block "header" . }}
+  {{ partialCached "header" . }}
+  {{ end }}
+  <div class="container standard-width mx-auto mt-5">
+    <h1>
+      {{ $collection.Title }}
+    </h1>
+    <div>
+      {{ $collection.Description | .RenderString }}
+    </div>
+    <div id="course-collection-container"></div>
+  </div>
+  <script>
+    {{ $collectionData := dict }}
+    {{ range $collection.Params.courses }}
+        {{ $url := delimit (slice "https://ocwnext.odl.mit.edu/courses/" .id "/data.json") "" }}
```

Comment:
sure, that sounds fine. that would let us basically use `data.json` from RC on an RC build, right?

Reply:
also, does the strategy of (essentially) curling down the JSON during the `ocw-www` build, storing it in a JSON literal in the rendered template, and loading it into a variable on `window` when the page actually parses and executes seem like a sound approach to you? I thought of a few other ways to do it, but this seemed the best to me because I think

1) we definitely don't want to load the json in the browser (some of these collections could have 30, 40, 50 or more courses in them, and the user would have to sit their waiting while we fetched them all, and we'd have to deal with CORS to make that work)
2) if a file isn't available or something then there will be a missing JSON error during the ocw-www build, which seems logical to me since we don't want to publish a course collection which references a course that it can't get data about

but wondering if you have any thoughts about it